Sources_v3/StreamChat/WebSocketClient/WebSocketClient_Tests.swift

```diff
@@ -273,9 +318,11 @@ class WebSocketClient_Tests: StressTestCase {
     }
     
     func test_webSocketPingController_ping_callsEngineWithPing() {
-        assert(engine.sendPing_calledCount == 0)
+        // Simulate connection
+        test_connectionFlow()
+
```

Comment:
`// Simulate connection to make sure web socket engine exists` ?

Reply:
fixed in c732d658ca9625bd74d35c32c4d80b3a70871a0e